src/neo/Ledger/Blockchain.cs

```diff
@@ -58,33 +56,12 @@ private class UnverifiedBlocksList { public LinkedList<Block> Blocks = new Linke
 
         private readonly static Script onPersistScript, postPersistScript;
         private const int MaxTxToReverifyPerIdle = 10;
-        private static readonly object lockObj = new object();
         private readonly NeoSystem system;
         private readonly IActorRef txrouter;
         private readonly Dictionary<UInt256, Block> block_cache = new Dictionary<UInt256, Block>();
         private readonly Dictionary<uint, UnverifiedBlocksList> block_cache_unverified = new Dictionary<uint, UnverifiedBlocksList>();
-        internal readonly RelayCache RelayCache = new RelayCache(100);
         private ImmutableHashSet<UInt160> extensibleWitnessWhiteList;
 
-        public IStore Store { get; }
-        /// <summary>
-        /// A readonly view of the blockchain store.
-        /// Note: It doesn't need to be disposed because the <see cref="ISnapshot"/> inside it is null.
-        /// </summary>
-        public DataCache View => new SnapshotCache(Store);
-        public MemoryPool MemPool { get; }
-        public HeaderCache HeaderCache { get; } = new HeaderCache();
-
-        private static Blockchain singleton;
-        public static Blockchain Singleton
```

Comment:
This file is quite clean now, much better.

And remove Singleton is also good in long term.